pytorch_lightning/plugins/training_type/single_tpu.py

```diff
@@ -75,9 +75,6 @@ def pre_dispatch(self, trainer: "pl.Trainer") -> None:
         self.tpu_local_core_rank = xm.get_local_ordinal()
         self.tpu_global_core_rank = xm.get_ordinal()
 
-    def save(self, state_dict: Dict, path: _PATH) -> None:
-        xm.save(state_dict, path)
-
```

Comment:
nit: this could be a separate PR

Reply:
noted in summary. it felt minor and relevant but happy to pull out to a separate PR